internals/cli/env.go

```diff
@@ -165,47 +158,211 @@ func (a *App) CheckStrictEnv() error {
 
 // CommandClause represents a command clause in a command0-line application.
 type CommandClause struct {
-	*kingpin.CmdClause
+	*cobra.Command
```

Comment:
If we decide to embed the `*cobra.Command` type in this struct we should make sure that we override all the methods we want to add functionality to, so we don't end up, for example, with both:
`cmd.CreateCommand` and `cmd.AddCommand`.

Alternatively we can have then cobra command as an unexported field and only expose the functions we want to allow ourselves to use.

Reply:
This has been solved.

Reply:
Can this conversation be resolved?